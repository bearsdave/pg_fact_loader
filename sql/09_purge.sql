SET client_min_messages TO warning;
--This is for testing functionality of timezone-specific timestamps
SET TIMEZONE TO 'America/Chicago';

SELECT COUNT(1) FROM test_audit_raw.customers_audit;

--We call this explicitly, because the worker will take the default add_interval of 1 hour, thus
--won't see any actual purging in the test suite.
SELECT fact_loader.purge_queues('0 seconds'::INTERVAL);

SELECT COUNT(1) FROM test_audit_raw.customers_audit;