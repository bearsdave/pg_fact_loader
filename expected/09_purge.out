SET client_min_messages TO warning;
--This is for testing functionality of timezone-specific timestamps
SET TIMEZONE TO 'America/Chicago';
SELECT COUNT(1) FROM test_audit_raw.customers_audit;
 count 
-------
    14
(1 row)

--We call this explicitly, because the worker will take the default add_interval of 1 hour, thus
--won't see any actual purging in the test suite.
SELECT fact_loader.purge_queues('0 seconds'::INTERVAL);
 purge_queues 
--------------
 
(1 row)

SELECT COUNT(1) FROM test_audit_raw.customers_audit;
 count 
-------
     4
(1 row)

